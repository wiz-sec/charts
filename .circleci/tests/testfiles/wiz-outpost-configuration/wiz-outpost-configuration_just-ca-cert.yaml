agent:
  clientId: client-id
  clientSecret: client-secret
  enabled: true
httpProxyConfiguration:
  caCertificate: --- BEGIN CERTIFICATE --- cert.me --- END CERTIFICATE ---
  enabled: true
  name: wiz-http-proxy-configuration
podAnnotations: null
