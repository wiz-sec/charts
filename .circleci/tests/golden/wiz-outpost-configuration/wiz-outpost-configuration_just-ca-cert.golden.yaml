---
# Source: wiz-outpost-configuration/templates/agent.secret.yaml
apiVersion: v1
kind: Secret
metadata:
  name: wiz-outpost-agent-creds
  labels:
    helm.sh/chart: "GOLDEN_STATIC_VALUE"
    app.kubernetes.io/name: wiz-outpost-configuration
    app.kubernetes.io/instance: release-test
    app.kubernetes.io/version: "0.0.1"
    app.kubernetes.io/managed-by: Helm
stringData:
  clientId: "client-id"
  clientSecret: "client-secret"
---
# Source: wiz-outpost-configuration/templates/httpProxyConfiguration.secret.yaml
apiVersion: v1
kind: Secret
type: Opaque
metadata:
  name: wiz-http-proxy-configuration
  labels:
    helm.sh/chart: "GOLDEN_STATIC_VALUE"
    app.kubernetes.io/name: wiz-outpost-configuration
    app.kubernetes.io/instance: release-test
    app.kubernetes.io/version: "0.0.1"
    app.kubernetes.io/managed-by: Helm
  annotations:
    meta.helm.sh/release-name: "release-test"
    meta.helm.sh/release-namespace: "release-helm-namespace"
data:
  caCertificate: "LS0tIEJFR0lOIENFUlRJRklDQVRFIC0tLSBjZXJ0Lm1lIC0tLSBFTkQgQ0VSVElGSUNBVEUgLS0t"
